### DISCUSSION TITLE
OpenAPI Chain Tool

### DISCUSSION THREAD

**Discussion by mdlmarkham** [FlowiseAI/Flowise#3231] [Q&A]
I would like to use the OpenAPI Chain Tool to allow a chat-bot to interact with my Jira account. I've generated the YML of the Jira OpenAPI spec and uploaded it to the flow. How do I configure the flow to use my API key for Jira?
![image](https://github.com/user-attachments/assets/95a76d91-22ba-4135-a0d2-1351f10498d3)

**Comment by mdlmarkham** [comment]
I'm ultimately trying to implement something like this: https://github.com/Vanals/Jira-tickets-with-ChatGPT using FloiwiseAI rather than a custom app.

**Comment by AnnoyingTechnology** [comment]
If anyone wants to chime in. 
Flowise's documentation for the OpenAPI tool is inexistant.
